### DISCUSSION TITLE
Clear old seats

### DISCUSSION THREAD

**Discussion by tudordanbalas** [onepub-dev/onepub#536] [Support]
Hello, 
When trying to remove some older colleagues from Onepub I get 
You cannot remove a member from their last organisation.
or 
You cannot remove the owner of an organisation

Is it possible to bypass that?

**Comment by bsutton** [comment]
Could you provide details of who you are trying to remove and from what
organisation.

**Comment by tudordanbalas** [comment]
Organisation name is <removed>.
I'm trying to remove <removed> and <removed>.
My email in the org is <removed>

**Comment by bsutton** [comment]
Tudor,
sorry for the late response - busy week.

I've disabled both Mike and Phil's members accounts which will free up
their seats.

Any problems let me know.

Brett

**Comment by bsutton** [comment]
I've also made you the owner.
